app/controllers/api/v1/sessions_controller.rb

```diff
@@ -1,11 +1,7 @@
 class Api::V1::SessionsController < ApplicationController
     skip_before_action :verify_authenticity_token
-    
+
     def new
-      user = User.find_by(username: params[:username])
```

Comment:
We did not need this functionality, we want a user to be able to visit login page for our app demonstration